cheat-sheet/content.md

```diff
@@ -0,0 +1,25 @@
+```python
+import mlflow
+import mflux_ai
+import mlflow.keras
+```
+_keras_ functions are the same for _sklearn_
+
+|Command|Help|
+|---|---|
+|**with mlflow.start_run() as run:**|exits after indent|
```

Comment:
Add an indented example code line below this to show that this is a context manager